### ISSUE TITLE
TouchableOpacity with accessibilityRole "tab" read "tab description" in English regardless of language settings

### ISSUE THREAD

**Issue by xboxdavchi** [facebook/react-native#30589]
## Description
For TouchableOpacity components that have an AccessibilityRole of "tab", for talkback and VoiceOver, they are always read as "tab description" in English. 

## React Native version:
0.63.3

## Steps To Reproduce
1. Create a touchable opacity with the prop "AccessibilityRole" of "tab". 
2. Change your device's language to any non-English language.
2. Navigate to the touchable opacity using talkback in Android or VoiceOver in iOS. 

## Expected Results
Talkback or VoiceOver should announce a localized string describing the "tab"-role touchable opacity.

## Actual Results
The non-localized, English string "tab description" is always announced.

**Comment by Waltari10** [comment]
I had the some problem and was able to fix it on Android. Here is the solution for Talkback: https://stackoverflow.com/questions/65560614/react-native-talkback-reads-component-accessibilityrole-in-wrong-language 

Basically in our project for Android we had to include the translations ourselves in file: `android/app/src/main/res/values/strings.xml:`

Like so: `<string name="button_description" translatable="false">Painike</string>`

List of strings to override is here: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/res/views/uimanager/values/strings_unlocalized.xml 

See StackOverflow link for more detail. I haven't been able to fix this on iOS though. Most accessibilityRole values seem to work there already.

**Comment by Waltari10** [comment]
Here is solution for iOS as well: https://valtterilaine.medium.com/react-native-accessibilityrole-i18n-support-for-ios-and-android-16d896e69178
